elements/pfe-page-status/src/pfe-page-status.scss

```diff
@@ -0,0 +1,78 @@
+@import "../../pfe-sass/pfe-sass";
+:host {
+  --pfe-status--bg-color: #{pfe-color(feedback--default)};
+  --pfe-status--text-color: #{pfe-color(feedback--default--lightest)};
```

Comment:
Instead of a separate text color variable - can we hook into the broadcast values instead and update the broadcast value to match the background?  So:
```
  --pfe-status--bg-color: #{pfe-color(feedback--default)};
  --pfe-broadcasted--color--text: #{pfe-color(feedback--default--lightest)};
```

Reply:
Also perhaps more BEM for the naming of the first variable?
```
--pfe-status--BackgroundColor: #{pfe-color(feedback--default)};
```